tests/Aql/FixedOutputExecutionBlockMock.cpp

```diff
@@ -0,0 +1,87 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2021-2021 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Michael Hackstein
+////////////////////////////////////////////////////////////////////////////////
+
+#include "FixedOutputExecutionBlockMock.h"
+
+#include "Aql/AqlCallStack.h"
+#include "Basics/voc-errors.h"
+
+using namespace arangodb;
+using namespace arangodb::aql;
+using namespace arangodb::tests::aql;
+
+namespace {
+// NOTE copy pasted from Waiting ExecutionBlock mock
```

Comment:
If it's copy-and-pasted, could it live somewhere to be used in both cases?

Reply:
I made them slightly different now.